cgi/user.pl

```diff
@@ -196,29 +196,28 @@
 
 		# Professional account
 		push @{$template_data_ref->{sections}}, {
-			id => "professional",
-			name => lang("pro_account"),
-			description => "if_you_work_for_a_producer",
-			note => "producers_platform_description_long",
-			fields => [
-				{
-					field => "pro",
-					type => "checkbox",
-					label => lang("this_is_a_pro_account"),
-					warning => sprintf(lang("this_is_a_pro_account_for_org"),"<b>" . $user_ref->{org} . "</b>"),
-					value => "off",
-				},
-				{
-					field => "pro_checkbox",
-					type => "hidden",
-					value => 1,
-				},
-				{
-					field => "requested_org",
-					label => lang("producer_or_brand") . ":",
-				}
-			]
-		};
+				id          => "professional",
```

Comment:
Could you revert the formatting to minimize the diff? I believe its because hash separators are aligned, might be helpful to adjust your editor's formatting settings.

Reply:
Thank you @ditsuke , I noticed that too!